src/components/common/CreateColonyWizard/TokenSelector.tsx

```diff
@@ -83,92 +60,54 @@ const getStatusText = ({
     };
   }
 
+  if (hasError || (!name && !symbol)) {
+    return {};
+  }
+
   return {
     status: MSG.preview,
     statusValues: { name, symbol },
   };
 };
 
+const getTokenData = (tokenQuery?: GetTokenByAddressQuery) => {
+  const tokenData = tokenQuery?.getTokenByAddress?.items;
+  const { name, symbol } = tokenData?.[0] || {};
+  return { name, symbol };
+};
+
+const getLoadingState = (isLoading: boolean) =>
+  // Don't show loading state if connection is 4G or unknown
+  connectionIs4G() !== false ? false : isLoading;
```

Comment:
The thought here is that on quick connections, the loading state is so brief that it looks better not to show it (else it flashes for a microsecond and then disappears).